projects/igniteui-angular/src/lib/grids/grid/grid.pinning.spec.ts

```diff
@@ -639,6 +675,65 @@ describe('IgxGrid - Column Pinning #grid', () => {
             expect(fistPinnedHeaders.classes['igx-grid__mrl-block']).toBeTruthy();
             expect(fistPinnedHeaders.classes['igx-grid__th--pinned-first']).toBeTruthy();
         });
+
+        it('should allow navigating to/from pinned area', (async() => {
+            pending('https://github.com/IgniteUI/igniteui-angular/pull/6910');
+            const fix = TestBed.createComponent(GridRightPinningComponent);
+            fix.detectChanges();
+            const grid = fix.componentInstance.instance as any;
+
+            const cellCompanyName = grid.getCellByColumn(0, 'CompanyName');
+            const range = { rowStart: 0, rowEnd: 0, columnStart: 9, columnEnd: 9 };
+            grid.selectRange(range);
+            grid.navigation.activeNode = {row: 0, column: 9};
+            fix.detectChanges();
+            expect(cellCompanyName.active).toBe(true);
+
+            const keydownArrowLeftEvent = new KeyboardEvent('keydown', {key: 'ArrowLeft'});
+            grid.navigation.dispatchEvent(keydownArrowLeftEvent);
+            await wait(DEBOUNCETIME);
+            fix.detectChanges();
+            const cellFax = grid.getCellByColumn(0, 'Fax');
+            expect(cellFax.active).toBe(true);
+            expect(cellCompanyName.active).toBe(false);
+
+            const keydownArrowRightEvent = new KeyboardEvent('keydown', {key: 'ArrowRight'});
+            grid.navigation.dispatchEvent(keydownArrowRightEvent);
+            await wait(DEBOUNCETIME);
+            fix.detectChanges();
+            expect(cellFax.active).toBe(false);
+            expect(cellCompanyName.active).toBe(true);
+        }));
+
+        it('should allow navigating to/from pinned area using Ctrl+Left/Right', (async() => {
+            pending('https://github.com/IgniteUI/igniteui-angular/pull/6910');
+            const fix = TestBed.createComponent(GridRightPinningComponent);
+            fix.detectChanges();
+            const grid = fix.componentInstance.instance as any;
+
+            const cellCompanyName = grid.getCellByColumn(0, 'CompanyName');
+            const range = { rowStart: 0, rowEnd: 0, columnStart: 9, columnEnd: 9 };
+            grid.selectRange(range);
+            grid.navigation.activeNode = {row: 0, column: 9};
+            fix.detectChanges();
+            expect(cellCompanyName.active).toBe(true);
+
+            const keydownArrowLeftEvent = new KeyboardEvent('keydown', {key: 'ArrowLeft', ctrlKey: true});
+            grid.navigation.dispatchEvent(keydownArrowLeftEvent);
```

Comment:
Same as the other you can use:
 grid.navigation.dispatchEvent(UIInteractions.getKeyboardEvent('keydown', 'ArrowDown', false, false, true))